CMakeLists.txt

```diff
@@ -398,7 +405,11 @@ endif (WIN32 AND HAVE_SNPRINTF)
 
 if (gflags_FOUND)
   target_include_directories (glog PUBLIC $<BUILD_INTERFACE:${gflags_INCLUDE_DIR}>)
-  target_link_libraries (glog PUBLIC ${gflags_LIBRARIES})
+  if (HUNTER_ENABLED)
+    target_link_libraries (glog PUBLIC gflags-static)
```

Comment:
Are you sure that the target is `gflags-static` and not `gflags`?

Reply:
I have to look into that with up to date gflags package

Reply:
TL:DR: there is a `gflags` target and also a `gflags_static` (notice the `_` instead of `-`) target in the default generated `gflags-config.cmake` file

with the default cmake configuration and installation (on Ubuntu 16.04) the target `gflags_static` is created by `gflags-targets.cmake`. In `gflags-config.cmake` a more convenient `gflags` import target is created

As for the `find_package` call the following is recommended in the config file:
```
# default settings of GFLAGS_SHARED and GFLAGS_NOTHREADS
#
# It is recommended to use either one of the following find_package commands
# instead of setting the GFLAGS_(SHARED|NOTHREADS) variables:
# - find_package(gflags REQUIRED)
# - find_package(gflags COMPONENTS nothreads_static)
# - find_package(gflags COMPONENTS nothreads_shared)
# - find_package(gflags COMPONENTS static)
# - find_package(gflags COMPONENTS shared)
```

We could explicitly call `find_package(gflags CONFIG COMPONENTS static)`, but static lib is the default